services/agents/postgresql.go

```diff
@@ -35,6 +35,11 @@ var postgresExporterAutodiscoveryVersion = version.MustParse("2.15.99")
 // postgresExporterConfig returns desired configuration of postgres_exporter process.
 func postgresExporterConfig(service *models.Service, exporter *models.Agent, redactMode redactMode,
 	pmmAgentVersion *version.Parsed) *agentpb.SetStateRequest_AgentProcess {
+
+	if service.DatabaseName == "" {
+		panic("database name not set")
```

Comment:
right now there is no error in function response, I think it is safer to fail fast

Reply:
Is it possible to have empty value here?

Reply:
if DB will not be available - it is a bug. Intention here is to show this explicitly and early.
Otherwise, connection string will be incorrect and error will be while establishing connection.